tests/unit/test_database.py

```diff
@@ -8,27 +8,27 @@
 from .fixtures import in_memory_db, redis_mongo_db  # noqa: F401
```

Comment:
Shouldn't this file be named test_database_public_methods.py?

Reply:
+1